resources/knative-eventing/Chart.yaml

```diff
@@ -1,9 +1,11 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
`v1` would make the existing Kyma operator happy as well as helm3 client. Let's go with that for now. Once the complete migration is done, we will switch to v2.

Reply:
For this PR, it means that the "requirements.yaml" file which is needed only by v1 should be restored. This is an incompatible change in v2, see also my comment below.